server/autotest_server/testers/ai/ai_tester.py

```diff
@@ -68,6 +70,16 @@ def call_ai_feedback(self) -> dict:
         timeout = test_group.get("timeout", 30)
         output_mode = test_group.get("output")
         cmd = [sys.executable, "-m", "ai_feedback"]
+
+        submission_file = config.get("submission")
+        if self._term_in_file(submission_file):
+            results[test_label] = {
+                "title": test_label,
+                "status": "success",
+                "message": "STUDENT OPTED OUT OF AI FEEDBACK",
```

Comment:
This doesn't need to be in all caps. You can use the message `"Student included the text "NO_EXTERNAL_AI_FEEDBACK" in their submission file. No AI feedback generated."`